emily/handler/src/database/entries/mod.rs

```diff
@@ -246,10 +246,21 @@ pub(crate) trait TableIndexTrait {
         // Convert inputs into the types needed for querying.
         let exclusive_start_key =
             maybe_exclusive_start_key_from_next_token::<Self::SearchToken>(maybe_next_token)?;
+
+        let table_name = Self::table_name(settings);
+        let index_name = format!("{:#?}", Self::INDEX_NAME_IF_GSI.map(|s| s.to_string()));
+        let partition_key_name =
+            <<Self::Entry as EntryTrait>::Key as KeyTrait>::PARTITION_KEY_NAME.to_string();
+        tracing::debug!(
+            %table_name,
+            %index_name,
+            %partition_key_name,
```

Comment:
I think you can do this:
```suggestion
        let index_name = Self::INDEX_NAME_IF_GSI;
        let partition_key_name = <<Self::Entry as EntryTrait>::Key as KeyTrait>::PARTITION_KEY_NAME;
        tracing::debug!(
            %table_name,
            %index_name,
            ?partition_key_name,
```
Or maybe remove the intermediate variables entirely. But no need to transform the `&str` into a `String`